source/opt/eliminate_dead_input_components_pass.cpp

```diff
@@ -25,9 +25,9 @@
 
 namespace {
 
-const uint32_t kAccessChainBaseInIdx = 0;
-const uint32_t kAccessChainIndex0InIdx = 1;
-const uint32_t kConstantValueInIdx = 0;
+constexpr uint32_t kAccessChainBaseInIdx = 0;
+constexpr uint32_t kAccessChainIndex0InIdx = 1;
+constexpr uint32_t kConstantValueInIdx = 0;
 
```

Comment:
Remove newlines to match format of the rest. It looks like when we only have constexpr variables we don't keep newlines